labelbox/client.py

```diff
@@ -931,6 +931,90 @@ def delete_unused_feature_schema(self, feature_schema_id):
                 "Failed to delete the feature schema, message: " +
                 response.json()['message'])
 
+    def update_feature_schema_definition(self, **kwargs):
+        feature_schema_id = kwargs.get("feature_schema_id")
+        title = kwargs.get("title")
+        endpoint = self.rest_endpoint + "/feature-schemas/" + feature_schema_id + '/definition'
+        response = requests.patch(
+            endpoint,
+            headers=self.rest_endpoint_headers,
+            json={"title": title},
+        )
+
+        if response.status_code == 200:
+            return response.json()
+        else:
+            raw_message = response.json()['message']
+            message = raw_message[0] if isinstance(raw_message,
+                                                   List) else raw_message
+            raise labelbox.exceptions.LabelboxError(
+                "Failed to update the feature schema's definition, message: " +
+                message)
+
+    def upsert_feature_schema(self, feature_schema):
+        """
+        Upserts a feature schema
+
+        Args:
+            feature_schema (FeatureSchema): The feature schema to upsert
+
+        Returns:
+            The upserted feature schema
+        """
+
+        feature_schema_id_to_upsert = feature_schema.get(
+            "featureSchemaId") or "new_feature_schema_id"
+        feature_schema_creation_endpoint = self.rest_endpoint + "/root-feature-schemas/" + feature_schema_id_to_upsert
```

Comment:
Why do we need a new REST resource `root-feature-schemas`? Could we use the same path as other API in `feature-schema.controller.ts`?